src/assetbundles/src/css/Translations.css

```diff
@@ -226,12 +226,6 @@
     align-self: center;
 }
 
-.utility-status .progressbar {
-    left: 50%;
-        width: 50%;
-    transform: translateX(-50%) translateY(-50%);
-}
-
```

Comment:
@bhupeshappfoster, instead of removing it, can we fix it by targeting it more specifically so that it doesn't interfere?

Reply:
@sidedwards The css removed is nowhere used in plugin code, so where and why do we target it?

Its also present in with exact element id, this one in use
![image](https://github.com/user-attachments/assets/5848c53f-d95b-4588-b08c-a01991f6f305)
 
 CC: @MerrittBoydston